docs: add hausanschluss bundles

ds-jhartmann opened the pull request:
<!-- 
Thanks for your contribution! 
Please follow the instructions on your PRs title and description.
aligned title description: '(feat|fix|chore|doc): _description of introduced change_'
Important: Contributing Guidelines can be found here: https://eclipse-tractusx.github.io/docs/oss/how-to-contribute
Info: <!- text comments ->  will be hidden from the rendered preview of your PR.
-->

## Description
<!-- 
Please describe your PR: 
- What does this PR introduce? 
- Does it fix a bug? 
- Does it add a new feature?
- Is it enhancing documentation?
-->

This pull request adds an install user guide for the hausanschluss bundles, and improves the chart README files.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

mgarciaLKS reviewed (approved):
Amazing job with the documentation, especially with the Guide, pretty straight forward to follow. Congratulations.

evegufy reviewed (changes requested):
**please only merge after 4th TX community days**, 26th of May should be fine

@saudkhan116 @mgarciaLKS @CDiezRodriguez @dvg36 please review

saudkhan116 commented:
The documentation is pretty good enough and easy to follow. I think, there is still one command missing to update the chart dependency, once a new or existing dependency is added/updated i.e., ``helm dependency update``.

evegufy reviewed (changes requested):
Please remove unnecessary empty lines and fix header pattern in all md files

ds-jhartmann commented on charts/dataspace-connector-bundle/README.md:
```diff
@@ -0,0 +1,120 @@
+
+## Values
+
+### Identity and Trust Protocol (IATP) Settings (Bring Your Own)
```
I was reusing the descriptions of the EDC here. Good to know that this naming is obsolete, I will change it.
`(Bring Your Own)` is intended to mark that this configuration can be replaced with your own wallet. This is described in the [hausanschluss bundle user guide](https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/chore/add-hausanschluss-documentation/docs/user/guides/hausanschluss-bundles.md#bring-your-own-components)

evegufy replied:
ok got it, would it make sense then to link to this section?

ds-jhartmann commented on charts/data-persistence-layer-bundle/README.md:
```diff
@@ -1,11 +1,52 @@
-# Data Persistence Layer Bundle
+# data-persistence-layer-bundle
```
The README is generated by the template. As Header it uses the name of the helm chart. But I agree, the name looks better like this. I will change it

ds-jhartmann commented:
Thank you for your reviews @evegufy and @saudkhan116
I added the `helm dependency update` and chart release process the user guide and fixed the formatting of the README templates and typos.
I would appreciate it, if you could take another look.

saudkhan116 commented:
> The documentation is pretty good enough and easy to follow. However, I found the following things still missing, which I think needs to be documented: Firstly, a command to update the chart dependency i.e. `helm dependency update` is not there. Secondly, if a bundle chart version updated, this should also reflect in the umbrella `Chart.yaml` file like as we do now for the tx-provider.

Hi @ds-jhartmann, Thank you to resolve my comments. I still don't find the second point in the docs. Could you please refer it here?

CDiezRodriguez reviewed (changes requested):
Looks amazing, @ds-jhartmann! I deployed it and data-exchange works fine. However, could you add the new `helm dependency update` step to the umbrella's local deployment? I noticed that some components have changed, and we now need to update more charts than just `tx-data-provider` and `umbrella`.

https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/main/docs/user/installation/local-repository.md#install-from-local-repository

evegufy commented on docs/user/guides/hausanschluss-bundles.md:
```diff
@@ -0,0 +1,240 @@
+## Introduction
+
+The **“Hausanschluss” Bundle** is a set of Helm Charts that simplifies deployment of all the Catena-X enablement
```
```suggestion
The **“Hausanschluss” Bundles** is a set of helm charts that simplifies deployment of all the Tractus-X enablement
```

evegufy commented on docs/user/guides/hausanschluss-bundles.md:
```diff
@@ -0,0 +1,240 @@
+
+Install a bundle with its default values:
+
+##### dataspace-connector-bundle
```
change to discussed header pattern

evegufy commented on docs/user/guides/hausanschluss-bundles.md:
````diff
@@ -0,0 +1,240 @@
+  --namespace umbrella --create-namespace
+```
+
+##### data-persistence-layer-bundle
````
change to discussed header pattern

evegufy commented on docs/user/guides/hausanschluss-bundles.md:
````diff
@@ -0,0 +1,240 @@
+  --set digital-twin-registry.registry.dataSource.url=jdbc:postgresql://dtr-postgresql:5432/dtr
+```
+
+##### identity-and-trust-bundle
````
change to discussed header pattern

evegufy commented on docs/user/guides/hausanschluss-bundles.md:
```diff
@@ -0,0 +1,240 @@
+
+### Bring-Your-Own Components
+
+Each Capability Bundle supports BYO of its dependencies:
```
```suggestion
Each Capability Bundle supports Bring-Your-Own (BYO) of its dependencies:
```

evegufy commented on docs/user/guides/hausanschluss-bundles.md:
````diff
@@ -0,0 +1,240 @@
+```
+
+The Bundles are pre-configured to work with in the Umbrella environment.  
+In case you want to use them in your own dataspace, you have to adjust the "Identity and Trust Protocol (IATP) Settings"
````
```suggestion
In case you want to use them in your own dataspace, you have to adjust the Decentralized Claims Protocol (formerly IATP) Settings"
```

evegufy commented on charts/dataspace-connector-bundle/values.yaml:
```diff
@@ -21,107 +21,210 @@
+  # === Bring Your Own ===
   iatp:
+    # -- Decentralized IDentifier (DID) of the connector
+    # @section -- Decentralized Claims Protocol (DCP) Settings (Bring Your Own)
```
please replace everywhere
```suggestion
    # @section -- Decentralized Claims Protocol (DCP, formerly IATP) Settings (Bring Your Own)
```

evegufy commented on docs/user/installation/local-repository.md:
````diff
@@ -1,13 +1,15 @@
+Update the chart dependencies.
 ```bash
-cd charts/umbrella/
+./hack/helm-dependencies.bash
````
In the install guide, I'm not a big fan of using the script in the hack directory, it's ok for workflows but here we wan't users to easily understand what's happening and using the script here makes it more hidden.
